RadarSDK/RadarAPIClient.m

```diff
@@ -496,4 +497,39 @@ - (void)ipGeocodeWithCompletionHandler:(RadarIPGeocodeAPICompletionHandler)compl
     }];
 }
 
+- (void)getContextWithLocation:(CLLocation * _Nonnull)location
+             completionHandler:(RadarGetContextAPICompletionHandler _Nullable)completionHandler {
+    NSString *publishableKey = [RadarSettings publishableKey];
+    if (!publishableKey) {
+        return completionHandler(RadarStatusErrorPublishableKey, nil, nil);
+    }
+
+    NSMutableString *queryString = [NSMutableString new];
+    [queryString appendFormat:@"coordinates=%.15f,%.15f", location.coordinate.latitude, location.coordinate.longitude];
+
+    NSString *host = [RadarSettings host];
+    NSString *url = [NSString stringWithFormat:@"%@/v1/context?%@", host, queryString];
+    
+    NSDictionary *headers = @{
+        @"Authorization": publishableKey,
+        @"Content-Type": @"application/json",
+        @"User-Agent": [RadarUtils userAgent],
+        @"X-Radar-Config": @"true",
+    };
```

Comment:
Not in the scope of this PR, but wonder if we could move these into some util function? I spot such headers several places in the codebase

Reply:
I think Nick has a PR that actually does this.. not sure if it's merged or a rebase is needed